arrow-libs/core/arrow-core-test/src/main/kotlin/arrow/core/test/laws/FxLaws.kt

```diff
@@ -23,38 +22,38 @@ object FxLaws {
   fun <Eff : Effect<*>, F, A> suspended(
     pureGen: Gen<F>,
     G: Gen<F>,
-    EQ: Eq<F>,
+    f: (F, F) -> Boolean,
```

Comment:
I'm wondering, wouldn't be easier replacing Eq as a typeclass for an internal typealias? That way you wouln't need to replace this everywhere. 

nit: Also we could name it the same way as before to indicate the behaviour hasn't change, currently is mixed, some places is `f`, some others is `eq` or `EQ`

Reply:
Yes, a lamda for it named `eq` would be a lot better. We can't use an internal typealias since we're exposing these laws to users.